locales/en-US.json

```diff
@@ -41,11 +44,14 @@
   "settings": "Settings",
   "shows": "Shows",
   "signIn": "Sign in",
+  "userSuccessfullySet": "User successfully set.",
   "unexpectedError": "Unexpected error",
   "unhandledException": "Unhandled exception",
   "upNext": "Up next",
+  "username": "Username",
   "filtersNotFound": "Unable to load filters",
   "status": "Status",
+  "passwordRequired": "Password Required",
   "preferedLanguage": "Prefered language",
```

Comment:
stick to one casing method for titles :p

Reply:
Bad habits :p